proposals/0293-extend-property-wrappers-to-function-and-closure-parameters.md

```diff
@@ -533,13 +613,13 @@ enum Review {
 switch Review(fromUser: "swiftUser5") {
 case .revised(@Traceable let reviewText),
      .original(let reviewText):
-  // do something with reviewText
+  // do something with 'reviewText'
 }
 ```
 
-### Property-wrapper parameters in memberwise initializers
+### Property-wrapper parameters in member-wise initializers
 
-Synthesized memberwise initializers could use property-wrapper parameters for stored properties with attached property wrappers:
+Synthesized member-wise initializers could use property-wrapper parameters for stored properties with attached property wrappers:
```

Comment:
The comment from the code example should be changed to this to reflect the change from "memberwise" to "member-wise":
```swift
// Synthesized member-wise init
```

Reply:
Heh, I made those changes because I did actually start using a spell checker. Of course, it doesn't spell check things in code blocks. Thanks!